### ISSUE TITLE
I have problems on calculating the loss function， the type does not match

### ISSUE THREAD

**Issue by chen-jing-315** [open-mmlab/mmclassification#817]
![图片](https://user-images.githubusercontent.com/84727107/167103462-0d7e700e-5b6b-4453-97e3-1356b2e43b08.png)

The following error occurred when I was training my dataset after modifying the configuration file. At first, I thought that it could not find the corresponding dataset label and could not convert it to the corresponding type. However,it still reported the same error after modifying the label, and then it report the following problem when debugging the mmcls /datasets/ voc.py.
![图片](https://user-images.githubusercontent.com/84727107/167104610-4534d114-821c-4d8d-a9ec-ccbec4c96c75.png)

**Comment by Ezra-Yu** [comment]
The import error may be that you have not selected the correct python interpreter.

Is the training dataset VOC or your own dataset? Have you tested on the VOC dataset, make sure that you can train on the VOC dataset correctly.

**Comment by tonysy** [comment]
This issue will be closed as it is inactive, feel free to re-open it if necessary.
